packages/node_modules/@webex/internal-plugin-devices/src/devices.js

```diff
@@ -1,6 +1,9 @@
 // Internal Dependencies
 import {WebexPlugin} from '@webex/webex-core';
 
+// Local Dependencies
```

Comment:
I updated this comment to be `Local` not `Internal`, as ESLint separates these on save.